src/workerd/api/actor-state.h

```diff
@@ -380,13 +380,30 @@ class DurableObjectState: public jsg::Object {
   // If no tag is provided, an array of all accepted WebSockets is returned.
   // Disconnected WebSockets are automatically removed from the list.
 
+  void setWebSocketAutoresponse(kj::String request, kj::String response);
+  // Sets an object-wide websocket auto response message for a specific
+  // request string. All websockets belonging to the same object must
+  // reply to request with response and store the request recived timestamp.
+
+  void unsetWebSocketAutoresponse();
+  // Unsets an object-wide websocket auto response.
+
   JSG_RESOURCE_TYPE(DurableObjectState, CompatibilityFlags::Reader flags) {
     JSG_METHOD(waitUntil);
     JSG_READONLY_INSTANCE_PROPERTY(id, getId);
     JSG_READONLY_INSTANCE_PROPERTY(storage, getStorage);
     JSG_METHOD(blockConcurrencyWhile);
     JSG_METHOD(acceptWebSocket);
     JSG_METHOD(getWebSockets);
+    JSG_METHOD(setWebSocketAutoresponse);
+    JSG_METHOD(unsetWebSocketAutoresponse);
+
+    if (flags.getWorkerdExperimental()) {
```

Comment:
Looks like this got added twice, maybe during a rebase fixing conflicts?

Reply:
Eh... Thanks for noticing, been rebasing this all week, somethings got duplicated often :upside_down_face: